crates/wasi/src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub use sync::*;
 #[cfg(feature = "tokio")]
 pub mod tokio {
     pub use wasi_tokio::*;
-    super::define_wasi!(async T: Send);
+    super::define_wasi!(async T: Send,);
```

Comment:
Not the prettiest fix, but I didn't manage to find a simple way to modify the macro to emit `,` if `bounds` are provided. Are there any suggestions?

Reply:
iirc this is a common macro-rules problem. Not a big deal, we only use the macro in this file.

Reply:
Found a better way by simply reordering them